docs/content/preview/tutorials/AI/azure-openai.md

```diff
@@ -194,7 +194,7 @@ With the Azure OpenAI models deployed and Airbnb data with embeddings loaded in
 
     ```sh
     cd {project_dir}/frontend
-    npm start
+    DANGEROUSLY_DISABLE_HOST_CHECK=true npm start
```

Comment:
Whst is this?

Reply:
Used as a workaround for this known [issue](https://github.com/facebook/create-react-app/issues/12304), just for the tutorial that we have.